drivers/scheduler/k8s/k8s.go

```diff
@@ -748,7 +748,8 @@ func (k *K8s) getAddressesForNode(n corev1.Node) []string {
 func (k *K8s) parseK8SNode(n corev1.Node) node.Node {
 	var nodeType node.Type
 	var zone, region string
-	if k8sCore.IsNodeMaster(n) {
+
+	if k8sCore.IsNodeMaster(n) && k.NodeDriverName != "ibm" {
```

Comment:
Do you think we can have this string `"ibm"` assigned to a const string?

Reply:
need to work on some tests for ibm specific, will update this along with the tests